eoscostarica/guias.eoscostarica.io
# Feat/plantilla readme

### GH Issue

Resolves #170  

### Steps to test
1. Check is under developer guidelines section
1. Check content is right

#### CheckList
